crates/matrix-sdk/src/widget/messages/actions/from_widget.rs

```diff
@@ -19,8 +19,9 @@ pub(crate) enum Action {
     SendEvent(MessageKind<SendEventRequest, SendEventResponse>),
     #[serde(rename = "org.matrix.msc2876.read_events")]
     ReadEvent(MessageKind<ReadEventRequest, ReadEventResponse>),
+    #[serde()]
+    Unknown,
```

Comment:
Hm, what is this used for?

Reply:
ugh that is a fragment of a previous idea.